README.md

```diff
@@ -301,86 +301,126 @@ ruff path/to/code/ --select F401 --select F403
 See `ruff --help` for more:
 
 ```shell
+<!-- Begin auto-generated cli help. -->
```

Comment:
It looks like these are appearing in the rendered Markdown.

Reply:
![Screen Shot 2022-12-30 at 1 30 31 PM](https://user-images.githubusercontent.com/1309177/210101734-3e81ef90-2ed4-4f50-9821-1882ae50a4af.png)

https://github.com/charliermarsh/ruff/blob/f3ebb168ef6d9ec5f1de59e0ea78a1fbf13c35f0/README.md

Reply:
:man_facepalming: Let me fix that.

Reply:
No prob, I wasn't sure what would happen 😂

Reply:
Fixed it.